ushahidi/platform-client
# Sms fix from qa

This pull request makes the following changes:
- Fixes from qa to targeted-surveys

Testing checklist:
- Go to the survey-list from settings
- Click on the text "Create a new survey" (not the button)
- [x] You should go to the create-new-survey-view
- Start creating a targeted survey
- When you come to adding questions
- Add a new question
- [x] There should be a cancel-button, not a delete-button
- Add a question that is more than 160 characters
- [x] The error-label should not cover the error-icon
- [x] The error-label should be correctly spelled
- Try saving
- [x] You should not be able to save
- [x] The error-message should remain
- shorten the text and save the question
- [x] You should be able to save
- Click on the question again to edit
- [x] There should be a delete-button
- Click the delete-button
- [x] The question should be deleted
- Add a question and continue
- [x] The text "Your telecommunications provider will charge you for each message. Please be careful about the costs you will incur by sending this survey. This action cannot be undone." should be visible in the alert
- [x] The publish-button should read "PUBLISH SURVEY"
- click on Publish survey
- [x] A modal should appear, the text "Your telecommunications provider will charge you for each message. Please be careful about the costs you will incur by sending this survey. This action cannot be undone" should appear in the modal
- Continue and publish survey
- [x] You should be redirected to survey-list-view
- [x] A notification-window should be visible

Fixes ushahidi/platform# .

Ping @ushahidi/platform
